pom.xml

```diff
@@ -44,6 +44,7 @@
     <modules>
         <module>modules/flink-ext</module>
         <module>modules/pub-sub</module>
+        <module>modules/zeromq-ext</module>
```

Comment:
Do we really need to add -ext to all of those modules' names?

Reply:
I had put `ext` to avoid conflict with existing released version. As we are planning to release from 1.0.0 version.